lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -56,7 +56,7 @@
   % endif
 <div class="course-container">
 <article class="course${mode_class}">
-  <% course_target = reverse(course_home_url_name(), args=[unicode(course_overview.id)]) %>
+  <% course_target = reverse(course_home_url_name(course_overview.id), args=[unicode(course_overview.id)]) %>
```

Comment:
I love this! It is a very concise way to determine the home page URL when it can vary per course.